core/src/main/java/org/bitcoinj/base/internal/TimeUtils.java

```diff
@@ -124,21 +122,9 @@ public static Duration longest(Duration duration1, Duration duration2) {
 
     /**
      * Formats a given date+time value to an ISO 8601 string.
-     * @param dateTime value to format, as a Date
+     * @param time date and time to format
      */
-    public static String dateTimeFormat(Date dateTime) {
-        DateFormat iso8601 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
-        iso8601.setTimeZone(UTC);
-        return iso8601.format(dateTime);
-    }
-
-    /**
-     * Formats a given date+time value to an ISO 8601 string.
-     * @param dateTime value to format, unix time (ms)
-     */
-    public static String dateTimeFormat(long dateTime) {
-        DateFormat iso8601 = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
-        iso8601.setTimeZone(UTC);
-        return iso8601.format(dateTime);
```

Comment:
Oh, you're not messing around! The old methods are going away!

Reply:
Yes, the API is internal, right?